pantheon-bundle/src/test/java/com/redhat/pantheon/servlet/module/ModuleVersionUploadTest.java

```diff
@@ -280,6 +280,139 @@ void uploadIdenticalDraftVersion() throws Exception {
+                .put("released", slingContext.resourceResolver().getResource("/new/module/es_ES/1").getValueMap().get("jcr:uuid"));
+
+        slingContext.resourceResolver().getResource("/new/module/es_ES/1").adaptTo(ModifiableValueMap.class)
+                .put("pant:hash", "bd7b5944327dce6ee8eb9573cb856d7528fbf8d634a4e8389a09f982571bf6c699f6dfefcd34fa6234e0acb19f46d1ee6d333a951476f1a712566bbc8d3552a2");
+        slingContext.resourceResolver().getResource("/new/module/es_ES/2").adaptTo(ModifiableValueMap.class)
+                .put("pant:hash", "cd7b5944327dce6ee8eb9573cb856d7528fbf8d634a4e8389a09f982571bf6c699f6dfefcd34fa6234e0acb19f46d1ee6d333a951476f1a712566bbc8d3552a2");
+
+        lenient().when(
+                asciidoctorService.getModuleHtml(
+                        any(ModuleVersion.class), any(Resource.class), anyMap(), anyBoolean()))
+                .thenReturn("A generated html string");
+        lenient().when(serviceResourceResolverProvider.getServiceResourceResolver())
+                .thenReturn(slingContext.resourceResolver());
+        ModuleVersionUpload upload = new ModuleVersionUpload(asciidoctorService, serviceResourceResolverProvider);
+        Map<String, Object> params = newHashMap();
+        params.put("locale", "es_ES");
+        params.put("asciidoc", "This is the draft adoc content");
+        registerMockAdapter(Module.class, slingContext);
+        slingContext.request().setParameterMap(params);
+        slingContext.request().setResource(new NonExistingResource(slingContext.resourceResolver(), "/new/module"));
+
+        // when
+        upload.doRun(slingContext.request(), new HtmlResponse(), null);
+
+        // Then
+        assertNotNull(slingContext.resourceResolver().getResource("/new/module/es_ES/2/content"));
+        assertNotNull(slingContext.resourceResolver().getResource("/new/module/es_ES/2/metadata"));
+        assertNotNull(slingContext.resourceResolver().getResource("/new/module/es_ES/1/content"));
+        assertNotNull(slingContext.resourceResolver().getResource("/new/module/es_ES/1/metadata"));
+
+        Module module =
+                SlingModels.getModel(slingContext.resourceResolver().getResource("/new/module"), Module.class);
+        assertEquals("This is the draft adoc content",
+                module.getDraftContent(LocaleUtils.toLocale("es_ES")).get().asciidocContent().get()
+        );
+        assertEquals("This is the draft html content",
+                module.getDraftContent(LocaleUtils.toLocale("es_ES")).get().cachedHtml().get().data().get()
+        );
+        assertEquals("This is the released adoc content",
+                module.getReleasedContent(LocaleUtils.toLocale("es_ES")).get().asciidocContent().get()
+        );
+        verify(asciidoctorService, never()).getModuleHtml(
+                any(ModuleVersion.class), any(Resource.class), anyMap(), anyBoolean());
+    }        @Test
```

Comment:
ooff... this formating :nauseated_face:

Reply:
Auto formatting sometimes does  strange things :)